docs/src/configuration/services/elasticsearch.md

```diff
@@ -32,6 +32,18 @@ To switch to OpenSearch, follow the same procedure as for [upgrading](#upgrading
 Elasticsearch 7.9 isn't available in the EU-1 and US-1 regions.
 Consider [region migration](../../guides/general/region-migration.md) if your project is in those regions.
 
+## Use a framework
+
+If you use one of the frameworks, follow the guide there:
+
+- [Drupal 9](../../guides/drupal9/elasticsearch.md)
```

Comment:
```suggestion
- [Drupal](../../guides/drupal9/elasticsearch.md)
```

It should also apply to 8, as far as I know, and given the approaching end of life, I wouldn't worry about 7.